wazo_acceptance/helpers/call_permission.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+class CallPermission:
+
+    def __init__(self, context):
+        self._context = context
+        self._confd_client = context.confd_client
+
+    def add_user(self, call_permission, user):
+        self._confd_client.call_permissions(call_permission).add_user(user)
+        self._context.add_cleanup(
+            self._confd_client.call_permissions(call_permission).remove_user,
+            user,
+        )
```

Comment:
FYI, since removing call permission removes users too, this helper is not mandatory.
But, I let you decide to keep it or not

Reply:
I'm removing it